style/ember/README.md

```diff
@@ -2,6 +2,7 @@ Ember
 =====
 
 * Don't put a space between the opening handlebars braces and the variable.
+* Don't use {{partial}} helper. Use a component with only a template.
```

Comment:
I think don't is a bit much here. There are a lot of times that partials make more sense because you're just grouping markup. I could get behind `prefer` here.